### PR TITLE
Update README to take not of k8s-migration package

### PR DESCRIPTION
**Pull request by ganeshmaharaj**
Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>

### CODE REVIEW

**Comment by ganeshmaharaj** [comment]
@mdhorn WDYT?

**Review by mdhorn — APPROVED** [review]
LGTM 👍

File: clr-k8s-examples/README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## Prerequisite
 This setup currently will work with Kubernetes 1.14 & above. Any version of Kubernetes before that might work, but is not guaranteed.
 
+## QUICK NOTE
+If you have an existing kubernetes cluster 1.17 & below on ClearLinux and would like to update to the latest version, please note a direct upgrade would not work. Follow the instructions provided at [k8s-migration](https://docs.01.org/clearlinux/latest/guides/clear/k8s-migration.html) in ClearLinux documentation.
```
Comment (mythi) [inline clr-k8s-examples/README.md:7]: Suggest: copy the the background section from the migration page and replace the first sentence with it.

Comment (ganeshmaharaj) [reply clr-k8s-examples/README.md]: @mythi  thanks and done.

**Review by mdhorn — APPROVED** [review]
LGTM 👍

**Review by mythi — APPROVED** [review]

**Review by jascott1 — APPROVED** [review]

**Pull request merged**